notebooks/preprocessing-workflow/tiling.ipynb

```diff
@@ -0,0 +1,292 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Tiling Utilities"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "This notebook contains a demo of the tiling utilities included in `IceFloeTracker.jl`. In particular, the following workflows are illustrated:\n",
+    "\n",
+    "- Getting tiling with tiles of a given size\n",
+    "- Getting the optimal tile size given an initial tile side length\n",
+    "\n",
+    "Run the cell below to set up the computation environment."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "HOME = \"../..\" # path to the root of the project two levels up\n",
+    "\n",
+    "# Activate the environment\n",
+    "using Pkg\n",
+    "Pkg.activate(HOME)\n",
+    "Pkg.precompile()\n",
+    "\n",
+    "using IceFloeTracker: get_tiles, get_tile_dims, load\n",
+    "using ColorTypes: RGBA"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "#### Load the image"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "img = load(\"bering_strait/truecolor/20200504.aqua.truecolor.250m.tiff\")"
```

Comment:
Should this point to the test images folder?

Reply:
Oh, I guess those images are not in this branch yet. I'll use one from the test folder, thanks!